From 1c03afc703fb6e398915e2b2b200b7db19b4e6b8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Mon, 26 Mar 2018 15:40:15 +0200
Subject: [PATCH 14/15] KCM: Fix typo in ccdb_sec_delete_list_done()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

When deleting the ccache we want to check if sec_key_list_len is equal 0
and not if sec_key_list is 0.

Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 src/responder/kcm/kcmsrv_ccache_secrets.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/responder/kcm/kcmsrv_ccache_secrets.c b/src/responder/kcm/kcmsrv_ccache_secrets.c
index 04dad9596..8a7a577d8 100644
--- a/src/responder/kcm/kcmsrv_ccache_secrets.c
+++ b/src/responder/kcm/kcmsrv_ccache_secrets.c
@@ -2007,7 +2007,7 @@ static void ccdb_sec_delete_list_done(struct tevent_req *subreq)
         return;
     }
 
-    if (sec_key_list == 0) {
+    if (state->sec_key_list_len == 0) {
         DEBUG(SSSDBG_MINOR_FAILURE, "No ccaches to delete\n");
         tevent_req_done(req);
         return;
-- 
2.14.3

